### PR TITLE
Add Intake and Roller classes

### PR DESCRIPTION
**Pull request by seadeef**
Adds classes for the intake and roller mechanisms on the robot. Updates constants, dimensions, and ports to reflect current conditions. Removes the `get_dist` method from the `Sensor` class because it is not needed. Changes the direction and behavior of some motors in order to maintain correct function on the robot.

### CODE REVIEW

File: src/Intake.cpp [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+	return m_on;
+}
+
+void Intake::set_on(bool value) {
+	m_on = value;
+	auto velocity = 0;
+	if (m_on) {
+		velocity = constants::INTAKE_VELOCITY;
+	}
+	m_left_motor.move_velocity(velocity);
```
Comment (Sptele) [inline src/Intake.cpp:17]: The way the code is implemented, this would only move on button press. Is this intended behavior?

Comment (seadeef) [reply src/Intake.cpp:17]: `move_velocity` runs in the background. Every time the button is pressed the motor velocity is toggled, which stays in effect until the button is pressed again. This is intended behavior.

Comment (Sptele) [reply src/Intake.cpp:17]: Oh yeah, I forgot that lol. Everything looks good to me!

**Review by Sptele — APPROVED** [review]
W

**Pull request merged**